src/Directory.Build.props

```diff
@@ -8,19 +8,20 @@
     <EnablePublicApi>true</EnablePublicApi>
 
     <!-- NuGet packages -->
+    <Version>0.6.0$(VersionSuffix)</Version>
```

Comment:
Shouldn't the NuGet package version end up to be `0.6.0-local.1`?

I'm a bit confused why you would still get `0.6.0` for the NuGet package here: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2286#discussion_r1124441886

Reply:
I mean that they show up with `AssemblyVersion` and `AssemblyFileVersion` as `0.6.0` but the NuGet package and the `AssemblyInformationlVersion` will be `0.6.0-local.1`. Here is what a user will see in Windows if they use the Explorer to check the version that came from the NuGet package:

![image](https://user-images.githubusercontent.com/5600755/223022078-a8c95244-9e15-460f-ad2d-b6ba70c90f3f.png)